code/graphics/2d.cpp

```diff
@@ -995,10 +995,8 @@ bool gr_init(std::unique_ptr<os::GraphicsOperations>&& graphicsOps, int d_mode,
 
 	if (gr_get_resolution_class(width, height) != GR_640) {
 		// check for hi-res interface files so that we can verify our width/height is correct
-		bool has_sparky_hi = (cf_exists_full("2_ChoosePilot-m.pcx", CF_TYPE_ANY) && cf_exists_full("2_TechShipData-m.pcx", CF_TYPE_ANY));
-
 		// if we don't have it then fall back to 640x480 mode instead
-		if ( !has_sparky_hi ) {
+		if ( !cf_exists_full("2_ChoosePilot-m.pcx", CF_TYPE_ANY)) {
```

Comment:
Mask files need to use PCX because the button indexes correspond to indexes in the PCX palette.  Visible files don't have this requirement so they can use any format.